mkosi/resources/man/mkosi.1.md

```diff
@@ -1336,7 +1336,7 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     | `xfsprogs`              | ✓      | ✓      | ✓      | ✓    | ✓      | ✓    | ✓        |
     | `xz`                    | ✓      | ✓      | ✓      | ✓    | ✓      | ✓    | ✓        |
     | `zstd`                  | ✓      | ✓      | ✓      | ✓    | ✓      | ✓    | ✓        |
-    | `zypper`                | ✓      |        | ✓      | ✓    | ✓      |      | ✓         |
+    | `zypper`                | ✓      |        | ✓      | ✓    | ✓      | ✓    | ✓        |
```

Comment:
You probably also want to add it to `mkosi/resources/mkosi-tools/mkosi.conf.d/10-arch.conf`

Reply:
Done! 😊